common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -195,6 +194,7 @@ def setUp(self):
         self.page = DiscussionTabHomePage(self.browser, self.course_id)
         self.page.visit()
 
+    @attr(shard=2)
```

Comment:
FYI @alisan617 @dianakhuang When a test class includes an a11y test point, the shard decorator needs to be on individual test points instead of at the class level. Otherwise the a11y test point runs twice-- once in regular bok choy test phase (here in shard 2), and once in the a11y test phase.